<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>주차정산 시스템</string>
  </property>
  <widget class="QLabel" name="img_plate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>460</width>
     <height>251</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>458</width>
      <height>249</height>
     </rect>
    </property>
    <widget class="QLabel" name="lbl_area">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>460</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="P0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>11</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>11</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>11</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>11</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="P6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="parking_info">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>460</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>630</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>요금조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pay_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>630</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>정   산</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_btn">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>630</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>종   료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>700</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>주차현황</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputText">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>660</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량번호</string>
   </property>
  </widget>
  <zorder>check_btn</zorder>
  <zorder>img_plate</zorder>
  <zorder>scrollArea</zorder>
  <zorder>parking_info</zorder>
  <zorder>lbl_title</zorder>
  <zorder>pay_btn</zorder>
  <zorder>end_btn</zorder>
  <zorder>search_btn</zorder>
  <zorder>InputText</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
